src/components/PlatformSwitcher.tsx

```diff
@@ -6,23 +6,36 @@ import { APPS_CONSTS } from '../constants/apps-consts'
 import useWidth from '../hooks/useWidth'
 import Icon from '@mdi/react'
 import { useDispatch } from 'react-redux'
-import { changeActiveApp } from '../redux/slices/app-config'
+import { changeActiveApp, getActiveApp } from '../redux/slices/app-config'
 import { useNavigate } from 'react-router-dom'
+import { useAppSelector } from '../hooks/reduxHooks'
+import { capitalize } from 'lodash'
+
+const DEFAULT_SELECTION_ITEM = {
```

Comment:
shouldn't this be moved to `apps-consts.ts` ?

Reply:
No it's not needed